etc/env/example.env

```diff
@@ -261,28 +263,28 @@ TYK_AUTH_API_SLUG=authentication
 TYK_CANDIG_API_ID=21
 TYK_CANDIG_API_NAME=CanDIG
 TYK_CANDIG_API_SLUG=candig
-TYK_CANDIG_API_TARGET=http://${CANDIG_DOMAIN}:${CANDIG_SERVER_PORT}
+TYK_CANDIG_API_TARGET=${CANDIG_PRIVATE_URL}
 TYK_CANDIG_API_LISTEN_PATH=candig
 
 ## api - katsu - chord-metadata
 TYK_KATSU_API_ID=31
 TYK_KATSU_API_NAME=Katsu
 TYK_KATSU_API_SLUG=katsu
-TYK_KATSU_API_TARGET=http://${CANDIG_DOMAIN}:${CHORD_METADATA_PORT}
+TYK_KATSU_API_TARGET=${CHORD_METADATA_PRIVATE_URL}
 TYK_KATSU_API_LISTEN_PATH=katsu
 
 ## api - mcode - candig-data-portal
 TYK_CANDIG_DATA_PORTAL_API_ID=41
 TYK_CANDIG_DATA_PORTAL_API_NAME="CanDIG Data Portal"
```

Comment:
i think we changed this to `candig-data-portal`

Reply:
as per discussions earlier lets use the same value for name and slug

also, i think we need to update our `policies.json` or its `tpl` to reflect the changes

Reply:
done